src/app/pages/services/components/service-dynamicdns/service-dynamicdns.component.ts

```diff
@@ -5,100 +5,116 @@ import * as _ from 'lodash';
 import { RestService, WebSocketService } from '../../../../services/';
 import { FieldConfig } from '../../../common/entity/entity-form/models/field-config.interface';
 import helptext from '../../../../helptext/services/components/service-dynamic-dns';
+import { FieldSets } from 'app/pages/common/entity/entity-form/classes/field-sets';
 
 @Component({
   selector : 'dynamicdns-edit',
   template : `<entity-form [conf]="this"></entity-form>`
 })
 
 export class ServiceDDNSComponent {
-  // protected resource_name = 'services/dynamicdns';
   protected addCall = 'dyndns.update';
   protected route_success: string[] = [ 'services' ];
 
-  public fieldConfig: FieldConfig[] = [
+  public fieldSets = new FieldSets([
     {
-      type : 'select',
-      name : 'provider',
-      placeholder : helptext.provider_placeholder,
-      tooltip: helptext.provider_tooltip,
-      options : []
+      name: helptext.fieldset_general,
+      label: true,
+      width: '49%',
+      config: [
+        {
+          type : 'select',
+          name : 'provider',
+          placeholder : helptext.provider_placeholder,
+          tooltip: helptext.provider_tooltip,
+          options : []
+        },
+        {
+          type : 'checkbox',
+          name : 'checkip_ssl',
+          placeholder : helptext.checkip_ssl_placeholder,
+          tooltip: helptext.checkip_ssl_tooltip,
+        },
+        {
+          type : 'input',
+          name : 'checkip_server',
+          placeholder : helptext.checkip_server_placeholder,
+          tooltip: helptext.checkip_server_tooltip,
+        },
+        {
+          type : 'input',
+          name : 'checkip_path',
+          placeholder : helptext.checkip_path_placeholder,
+          tooltip: helptext.checkip_path_tooltip,
+        },
+        {
+          type : 'checkbox',
+          name : 'ssl',
+          placeholder : helptext.ssl_placeholder,
+          tooltip: helptext.ssl_tooltip,
+        },
+        {
+          type: 'input',
+          name: 'custom_ddns_server',
+          placeholder: helptext.custom_ddns_server_placeholder,
+          tooltip: helptext.custom_ddns_server_tooltip,
+          disabled: true,
+          isHidden: true
+        },
+        {
+          type: 'input',
+          name: 'custom_ddns_path',
+          placeholder: helptext.custom_ddns_path_placeholder,
+          tooltip: helptext.custom_ddns_path_tooltip,
+          disabled: true,
+          isHidden: true
+        },
+        {
+          type : 'input',
+          name : 'domain',
```

Comment:
it complains about domain name being a required field, if this is in fact required now we should probably mark it with required:true so the user cannot save without passing this validation